prime-router/src/main/kotlin/SettingsProvider.kt

```diff
@@ -44,19 +45,11 @@ enum class CustomerStatus {
  * A submission with topic FULL_ELR will be processed using the full ELR pipeline (fhir engine), submissions
  * from a sender with topic COVID_19 will be processed using the covid-19 pipeline.
  */
-enum class Topic(val json_val: String) {
-    @JsonProperty("full-elr")
-    FULL_ELR("full-elr"),
-
-    @JsonProperty("covid-19")
-    COVID_19("covid-19"),
-
-    @JsonProperty("monkeypox")
-    MONKEYPOX("monkeypox"),
-
-    @JsonProperty("CsvFileTests-topic")
-    CSV_TESTS("CsvFileTests-topic"),
-
-    @JsonProperty("test")
-    TEST("test"),
+enum class Topic(@JsonValue val jsonVal: String, val isUniversalPipeline: Boolean = true) {
```

Comment:
Not sure where to put this comment, but with this implementation we are limiting ourselves to two pipelines... universal and not universal. Have we thought about the implications here? This design is not open-closed to new pipelines. Does it need to be? Not sure...

Reply:
For example, what about when we start processing case data? I don't think we know for sure yet, and so it's better we err on the side of good practices, in my opinion, and not box ourselves in.

Reply:
Fair question! My take is that at least in the short-term future, we can foresee two (one legacy, one universal) pipelines with the just the pre-existing topics on legacy, and at least two (potentially many more) topics on universal. So I think this setup serves us well for that short-term. I think it'd be easy enough to remove the `isUniversalPipeline` if we ever go down to just one pipeline, or if we need to extend to other pipelines, we can fairly simply replace `isUniversalPipeline` with some kind of `pipeline` variable (either on `topic` or separately from `topic` if those do get decoupled). Good to think about the implications, but I don't think this implementation locks us too hard into anything, and without knowing what that might look like in the far future, I'm not sure what we would want to do differently.